src/testdrive/src/error.rs

```diff
@@ -169,6 +169,12 @@ impl From<InputError> for Error {
     }
 }
 
+impl From<String> for Error {
+    fn from(err: String) -> Error {
+        Error::message(err)
+    }
+}
```

Comment:
`Error::message("something happened")` seems more clear to me than `"blah".to_string()` and is almost exactly as concise.

Reply:
Yeah here this is better, I had this lying around because we have other places where this would be much more convenient (a bunch of `wrap_err` lambda calls) but we can ditch that stuff when we migrate to anyhow.